library/src/morpheus-master/sass/base/_defaults.scss

```diff
@@ -215,6 +215,20 @@ button[class="disabled"]:hover,
 	text-decoration: none;
 }
 
+input[type="checkbox"].formButtonDisabled{
+	@include border-radius(3px);
+	&:checked{
+		background-image: url("images/check.png") !important;
```

Comment:
Unfortunately I won't have the time to invest in untangling the existing CSS issues for several more months, as we're in the midst of a massive upgrade to 11.

I decided to contribute this because it's very small, and the issue is currently affecting not only 12, but also any institutions using 11. It makes things look very bad. Steps to reproduce in the JIRA ticket.